src/python_testing/TC_RVCOPSTATE_2_3.py

```diff
@@ -80,74 +80,181 @@ async def test_TC_RVCOPSTATE_2_3(self):
+            operational_state = await self.read_mod_attribute_expect_success(endpoint=self.endpoint,
+                                                                             attribute=attributes.OperationalState)
+            logging.info("OperationalState: %s" % operational_state)
+            asserts.assert_equal(operational_state, Clusters.OperationalState.Enums.OperationalStateEnum.kRunning,
+                                 "OperationalState(%s) should be Running(0x01)" % operational_state)
+
+            self.print_step(18, "Manually cause the device to pause running due to an error, and be able to resume after clearing the error")
+            input("Press Enter when done.\n")
+
+            self.print_step(19, "Read OperationalState attribute")
+            operational_state = await self.read_mod_attribute_expect_success(endpoint=self.endpoint,
+                                                                             attribute=attributes.OperationalState)
+            logging.info("OperationalState: %s" % operational_state)
+            asserts.assert_equal(operational_state, Clusters.OperationalState.Enums.OperationalStateEnum.kError,
+                                 "OperationalState(%s) should be Error(0x03)" % operational_state)
+
+            self.print_step(20, "Manually clear the error")
+            input("Press Enter when done.\n")
+
+            self.print_step(21, "Read OperationalState attribute")
+            operational_state = await self.read_mod_attribute_expect_success(endpoint=self.endpoint,
+                                                                             attribute=attributes.OperationalState)
+            logging.info("OperationalState: %s" % operational_state)
+            asserts.assert_equal(operational_state, Clusters.OperationalState.Enums.OperationalStateEnum.kPaused,
+                                 "OperationalState(%s) should be Paused(0x02)" % operational_state)
+
+            self.print_step(22, "Send Resume command")
+            ret = await self.send_resume_cmd()
+            asserts.assert_equal(ret.commandResponseState.errorStateID, Clusters.OperationalState.Enums.ErrorStateEnum.kNoError,
+                                 "errorStateID(%s) should be NoError(0x00)" % ret.commandResponseState.errorStateID)
+
+            self.print_step(23, "Read OperationalState attribute")
+            operational_state = await self.read_mod_attribute_expect_success(endpoint=self.endpoint,
+                                                                             attribute=attributes.OperationalState)
+            logging.info("OperationalState: %s" % operational_state)
+            asserts.assert_equal(operational_state, Clusters.OperationalState.Enums.OperationalStateEnum.kRunning,
+                                 "OperationalState(%s) should be Running(0x01)" % operational_state)
+
+        self.print_step(24, "Manually put the device in the Stopped(0x00) operational state")
```

Comment:
Each of these manual steps, and the testing that follows them, needs to guarded by the respective PICS code.